code/src/components/ArtistName.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react' 
+
+const ArtistName = (props) => {
+
+    console.log(props)
```

Comment:
If you want you can remove the console.log() here - but if you think it can be useful for people viewing the page then of course you can keep it!